testdata/batch-submitter/configMap.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: batch-submitter
+data:
+  BUILD_ENV: development
+  ETH_NETWORK_NAME: hardhat
+  LOG_LEVEL: debug
+
+  L1_ETH_RPC: "http://l1chain-svc:8545"
```

Comment:
We have to change it later. If we fix the l1 rpc, let me know!

Reply:
Yes! thank you